setup.py

```diff
@@ -27,7 +27,8 @@
     'fabric==1.8.3',
     'wagon==0.3.0',
     'fasteners==0.13.0',
-    'pyzmq==15.1.0'
+    'pyzmq==15.1.0',
+    'virtualenv==15.0.1'
```

Comment:
Considering that `wagon` has this dependency: `"virtualenv>=12.1"`, perhaps it'd be better to have the same range here instead of the hard coded `15.0.1`